### PR TITLE
add Majority Element

### PR DESCRIPTION
**Pull request by shivamp1998**
[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### Describe your change:

- [x] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### Checklist:

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x] All new JavaScript files are placed inside an existing directory.
- [x] All filenames should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
- [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: Data-Structures/Array/MajorityElement.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            return 1;
+        }
+    }
+    return 0;
+}
+
+function Majority(array) {
+    let size = array.length;
+    //finds the candidate for majority
+    let cand = Candidate(array,size);
```
Comment (raklaptudirm) [inline Data-Structures/Array/MajorityElement.js:48]: Candidate takes only one input.

**Review by raklaptudirm — CHANGES_REQUESTED** [review]
Format your code using standard.js

File: Data-Structures/Array/MajorityElement.js (hunk 1 above)
Comment (shivamp1998) [reply Data-Structures/Array/MajorityElement.js]: Formatted code using standard.js, and fixed paramters of Candidate function.

**Review by appgurueu — CHANGES_REQUESTED** [review]
Do not remove [`package-lock.json`](https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json)

**Comment by appgurueu** [comment]
> > Do not remove [`package-lock.json`](https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json)
> 
> package-lock.json is in my commit, i have not removed it.

Bildschirmfoto von 2022-10-06 17-13-24

**Comment by shivamp1998** [comment]
Will create a new pull request.

**Pull request closed without merging**